src/parser/condition.rs

```diff
@@ -184,7 +194,7 @@ fn build_condition_recursively(name: &str, obj: &Value) -> Result<ConditionValue
                 };
                 ConditionValue::Condition(condition_name.to_string())
             }
-            "!Ref" | "Ref" => {
+            Some("Ref") => {
```

Comment:
Same for Ref here: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-ref.html#ref-example